src/main/java/com/puppycrawl/tools/checkstyle/JavadocDetailNodeParser.java

```diff
@@ -478,13 +478,13 @@ public void syntaxError(
                 errorMessage = new ParseErrorMessage(lineNumber,
                         MSG_JAVADOC_MISSED_HTML_CLOSE, charPositionInLine, token.getText());
 
-                throw new ParseCancellationException(msg);
+                throw new IllegalArgumentException(msg);
```

Comment:
Nevermind, its an overridden method and `ParseException` isn't a runtime exception.